docs/changelog.yaml-format.md

```diff
@@ -174,13 +174,13 @@ Currently valid plugin types are:
 10. `shell`
 11. `strategy`
 12. `vars`
-13. `filter` (not yet documentable by `ansible-doc`)
-14. `test` (not yet documentable by `ansible-doc`)
+13. `filter` (documentable by `ansible-doc` from ansible-core 2.14+, or the current devel and milestone versions)
+14. `test` (documentable by `ansible-doc` from ansible-core 2.14+, or the current devel and milestone versions)
 
 See `DOCUMENTABLE_PLUGINS` in https://github.com/ansible/ansible/blob/devel/lib/ansible/constants.py for a complete list of plugin types (minus `modules`).
 
 Currently valid object types are:
-1. `role`
+1. `role` (documentable by `ansible-doc` from ansible-core 2.11+)
```

Comment:
might want to mention that this only works when an argument spec is present

EDIT: I see this is mentioned in more detail in `changelogs.rst`, but I think it's worth mentioning here, because it will otherwise throw people off I'm betting ("why isn't my role documentation being generated?!")

Reply:
This document describes the format of changelog.yaml (so that folks who create this file manually, or from another program than antsibull-changelog, have a reference on how it should look like). So I don't think that should really be in here. (I'm even wondering whether the note on ansible-doc should be removed.)